docs/rules/require-expect.md

```diff
@@ -11,6 +11,18 @@ test. This rule checks for `expect` at linting time.
 
 ## Options
 
+This rule is configurable by  the `Check Type` and the `Config Options`
+
+### Check Type
+
+Example:
+
+```js
+rules: {
```

Comment:
In order for eslint-plugin-markdown to pass, please surround this object with curly braces and remove the dangling comma at the end. Thanks!